common/app/views/fragments/header.scala.html

```diff
@@ -9,112 +9,122 @@
 @* TODO: #new-recipe - remove when removing the test *@
 @hasSlimHeader = @{page.metadata.hasSlimHeader || (page.metadata.isNewRecipeDesign && mvt.ABNewRecipeDesign.isParticipating)}
 
-<header id="header"
-        class="l-header u-cf @if(hasSlimHeader && !showNormalHeader) {l-header--is-slim l-header--no-navigation} js-header hide-on-mobile"
-        role="banner"
-        data-link-name="global navigation: header">
-    <div class="js-navigation-header navigation-container navigation-container--collapsed">
-        <div class="gs-container l-header__inner">
-            <div class="l-header-pre u-cf">
-                <div class="brand-bar">
-                    @if(IdentityProfileNavigationSwitch.isSwitchedOn) {
-                        <div class="brand-bar__item brand-bar__item--profile popup-container
+@if(!mvt.ABNewDesktopHeader.isParticipating) {
```

Comment:
Yeah, wish I had seen this comment before scanning the whole file for diffs.  Quite a chunky template that one...